management/server/user.go

```diff
@@ -91,10 +94,13 @@ func (u *User) toUserInfo(userData *idp.UserData) (*UserInfo, error) {
 func (u *User) Copy() *User {
 	autoGroups := make([]string, 0)
```

Comment:
When you create a new slice try to set a proper capacity. The append is more efficient in this case. I think in this scenario the built in copy function fit better here.